sos/plugins/ceph.py

```diff
@@ -22,10 +23,6 @@ class Ceph(Plugin, RedHatPlugin, UbuntuPlugin):
     plugin_name = 'ceph'
     profiles = ('storage', 'virt')
 
-    option_list = [
-        ("log", "gathers all ceph logs", "slow", False)
-    ]
-
```

Comment:
There's something odd going on with this branch. I only see one commit on it in GitHub, but it's changing lines that don't exist in master (there is no `log` option in `ceph.py`).

The changes look fine now - if you can sort out the problem with the branch (combining commits or rebasing), I'll get it merged.